File: app/src/main/java/com/jlillioja/press/database/PressDatabase.java
package com.jlillioja.press.database;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by Jacob on 12/22/2016.
 */

@Database(name = PressDatabase.NAME, version = PressDatabase.VERSION)
public class PressDatabase {
    public static final String NAME = "MyDataBase";

    public static final int VERSION = 1;
}

File: app/src/main/java/com/jlillioja/press/PressDatabase.kt
package com.jlillioja.press

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Created by Jacob on 12/22/2016.
 */

@Database(name = PressDatabase.NAME, version = PressDatabase.VERSION)
object PressDatabase {

    const val NAME = "PressDatabase" // we will add the .db extension
    const val VERSION = 1

}